src/Jellyfin.Database/Jellyfin.Database.Providers.Sqlite/Migrations/20250401142247_FixAncestors.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Jellyfin.Server.Implementations.Migrations
+{
+    /// <inheritdoc />
+    public partial class FixAncestors : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Why is there an empty migration?

Reply:
That's what EFcore created, can I just remove it?

Reply:
I think it's because the navigation properties aren't stored in the db, but in config / snapshot. I would leave this empty migration in for tracking?